### PR TITLE
Done the breadcrumb

### PR DESCRIPTION
**Pull request by diegomateus**

### CODE REVIEW

File: src/views/partials/breadcrumb.ejs [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<nav class="breadcrumb-navigation">
+    <ul>
+        <li class="home-link" >
+            <a href="../">home</a>
```
Comment (monotera) [inline src/views/partials/breadcrumb.ejs:4]: It should just be `href="/"`

File: src/static/css/breadcrumb.scss [hunk 2]
```diff
@@ -0,0 +1,53 @@
+.breadcrumb-navigation {
+    padding-left: 15px;
+    
+    ul {
```
Comment (pushdev-code) [inline src/static/css/breadcrumb.scss:4]: use classes instead of tag names.

File: src/static/css/breadcrumb.scss [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+        display: flex;
+        font-size: 14px;
+        list-style: none;
+        margin-top: 150px;
+        padding: 12px;
+
+        li {
```
Comment (pushdev-code) [inline src/static/css/breadcrumb.scss:11]: use classes

File: src/static/css/breadcrumb.scss [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+            padding-left: .5rem;
+
+            a {
```
Comment (pushdev-code) [inline src/static/css/breadcrumb.scss:15]: use classes

File: src/views/pages/product.ejs [hunk 5]
```diff
@@ -7,10 +7,15 @@
 <body>
     <section class="main-container">
         <%- include('../partials/header') %>
-        <%- include('../partials/gallery')%>
+        <div>
```
Comment (pushdev-code) [inline src/views/pages/product.ejs:10]: can this div be removed?

Comment (diegomateus) [reply src/views/pages/product.ejs:10]: This is the same file of the Product detail, in this div should be the include of the information of the product

**Pull request merged**